src/registrar/templates/401.html

```diff
@@ -25,7 +26,7 @@ <h2>
         Would you like to <a href="{% url 'login' %}"> try logging in again?</a>
       </p>
       <p>
-        If you would like help with this error <a href="https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/site/cisagov/getgov-home/contact/"> contact us </a>
+     If you'd like help with this error <a href="{% public_site_url 'contact' %}"> contact us </a>.
```

Comment:
**question (non-blocking)**: I think that this probably works either way, but the new link won't have a trailing slash. Was that intentional?

Reply:
It wasn't intentional. Is it a better practice to include the slash?